base/applications/rapps/include/rapps.h

```diff
@@ -13,4 +13,7 @@
 #include "misc.h"
 #include "configparser.h"
 
+extern LONG g_Busy;
```

Comment:
Only one single thread (no concurrent ones) is using this variable?

Reply:
Correct. Will change to Interlocked access if this changes.